ui/components/app/assets/nfts/nfts-tab/nfts-tab.tsx

```diff
@@ -118,10 +118,12 @@ export default function NftsTab() {
   };
 
   const onRefresh = () => {
+    trace({ name: TraceName.NftsTabRefresh });
```

Comment:
Can traces also be used in the background/service worker?

Since we aren't awaiting anything here, I feel like the trace will immediately complete?